contracts/ethereum/SnapshotXZodiacModule/SnapshotXL1Executor.sol

```diff
@@ -53,15 +56,15 @@ contract SnapshotXL1Executor is Module, SnapshotXProposalRelayer {
    * @param _owner Address of the owner of this contract
    * @param _avatar Address that will ultimately execute function calls
    * @param _target Address that this contract will pass transactions to
-   * @param _decisionExecutorL2 Address of the StarkNet contract that will send execution details to this contract in a L2 -> L1 message
+   * @param _l2ExecutionRelayer Address of the StarkNet contract that will send execution details to this contract in a L2 -> L1 message
    * @param _starknetCore Address of the StarkNet Core contract
    */
   event SnapshotXL1ExecutorSetUpComplete(
```

Comment:
should we add the whitelisted spaces to this event?

Reply:
@bonustrack do you need the set of whitelisted spaces emitted too?

Reply:
lets leave for now, can add in future